CR2WTests/StressTest.cs

```diff
@@ -7,487 +7,598 @@
 using System.IO;
 using WolvenKit.CR2W;
 using WolvenKit.Bundles;
+using WolvenKit.Common;
 using System.Threading.Tasks;
+using System.Diagnostics;
+using System.Text;
+using System.Security.Cryptography;
+using WolvenKit.Common.Extensions;
 
 namespace CR2WTests
 {
+    public static class AssertEx
+    {
+        public static async Task<TException>
+          ThrowsAsync<TException>(Func<Task> action,
+          bool allowDerivedTypes = true) where TException : Exception
+        {
+            try
+            {
+                await action();
+            }
+            catch (Exception ex)
+            {
+                if (allowDerivedTypes && !(ex is TException))
+                    throw new Exception("Delegate threw exception of type " +
+                      ex.GetType().Name + ", but " + typeof(TException).Name +
+                      " or a derived type was expected.", ex);
+                if (!allowDerivedTypes && ex.GetType() != typeof(TException))
+                    throw new Exception("Delegate threw exception of type " +
+                      ex.GetType().Name + ", but " + typeof(TException).Name +
+                      " was expected.", ex);
+                return (TException)ex;
+            }
+            throw new Exception("Delegate did not throw expected exception " +
+              typeof(TException).Name + ".");
+        }
+        public static Task<Exception> ThrowsAsync(Func<Task> action)
+        {
+            return ThrowsAsync<Exception>(action, true);
+        }
+    }
+
+
     [TestClass]
     public class StressTest
     {
         static BundleManager mc;
+        static Dictionary<string, MemoryMappedFile> mmfs;
 
         [ClassInitialize]
         public static void Setup(TestContext context)
         {
+            mmfs = new Dictionary<string, MemoryMappedFile>();
             mc = new BundleManager();
-            mc.LoadAll("D:\\SteamLibrary\\steamapps\\common\\TW3\\bin\\x64");
-            //mc.LoadAll("C:\\Steam\\steamapps\\common\\The Witcher 3\\bin\\x64");
+            //mc.LoadAll("D:\\SteamLibrary\\steamapps\\common\\The Witcher 3\\bin\\x64");
```

Comment:
We should probably change this so we don't have to edit it in every commit. How about %TW3_DIR%/bin/x64?